about_us.php

```diff
@@ -0,0 +1,130 @@
+            <p class="m-0">Home / About Us 01</p>
+        </div>
+    </div>
+</div>
+<div class="d-flex justify-content-center mt-5">
+</div>
+<h1 class="text-center mt-3">Welcom to Auteur</h1>
+<h3 class="text-center mt-3">A MONTHLY BOOKS REVIEW PUBLICATION</h3>
+<div class="container mt-5">
+    <p>Imagine two men facing each other, pointing past one another. One is pointing at a tornado that is coming,
+        and the other at a raging fire headed towards them. Each sees their own truth and is angry at the sight of
+        the other’s hand. Each feels
+        that the other’s hand is “wrong.” This may seem silly, but replace the tornado and fire with any modern
+        issues, and the hands with words, and this scene describes how we often try to communicate.</p>
+    <div class="d-flex mt-5">
+        <h3 class="ms-5">We point past each other with our words, arguing as though we are looking at the same facts
+            and experiences.</h3>
+    </div>
+    <p class="mt-5">We want to prove our words are the right ones, instead of learning to look at what the other’s
+        words are pointing at. Words are seductive, and for all their undeniable usefulness, they also can lead us
+        away from understanding when we focus on
+        them, when we make them more important than the truth they are meant to point at.</p>
+</div>
+<div class="container">
+    <video src="https://www.youtube.com/watch?v=eH2WNtL5ong "></video>
+    <div class="container-fluid mb-5 ">
+        <div class="row ">
+            <div class="col-sm-4 mt-4">
+                <img src="assets/images/single-image-16.jpg " width="100% " alt="Error:404">
+            </div>
+            <div class="col-sm-4 mt-4 ">
+                <img src="assets/images/single-image-18.jpg " width="100% " alt="Error:404">
+            </div>
+            <div class="col-sm-4 mt-4">
+                <p>Lorem ipsum dolor sit amet, consectetur cing elit. Suspe ndisse suscipit sagittis leo sit
+                    estibulum issim Lorem ipsum dolor sit amet, AKON lsicn íoifa lkoix consectetur cing elit.</p>
+                <p class="fw-bold ">ADDRESS</p>
+                <i>1800 Abbot Kinney Blvd. Unit D & E Venice</i>
+                <p class="fw-bold ">PHONE</p>
+                <i>Mobile: (+88) – 1990 – 6886</i>
+                <p><i>Hotline: 1800 – 1102</i></p>
+                <p class="fw-bold ">Email</p>
+                <p>contact@auteur.com</p>
+            </div>
+        </div>
+    </div>
+</div>
+<div class="container-fluid " style="background-color: rgba(211, 211, 211, 0.253); ">
+    <h3 class="text-danger text-center ">TESTIMONIALS</h3>
+    <h1 class="text-center ">Read Reviews by My Readers</h1>
+    <div class="container ">
+        <div class="row ">
+            <div class="col-sm-6 mt-5 ">
+                <img class="rounded-circle " src="assets/images/testimonial-01-140x140.jpg " alt="Error:404">
```

Comment:
remove Error:404 and add image not found

Reply:
follow this instruction